src/en/users/04-votes-and-ranking.md

```diff
@@ -0,0 +1,28 @@
+## Votes and ranking
+
+Lemmy uses a voting system to sort post listings. On the left side of each post are up and down arrows, which let you *upvote* or *downvote* it. You can upvote posts that you like so that more users will see them. Or downvote posts so that they are less likely to be seen. Each post receives a score which is the number of upvotes minus number of downvotes.
+
+When browsing the frontpage or a community, you can choose between the following sort types for posts:
+
+- **Active** (default): Calculates a rank based on the score and time of the latest comment, with decay over time. See [this page] for more details
+- **Hot**: Like active, but uses time when the post was published
+- **New**: Shows most recent posts first
+- **Old**: Shows oldest posts first
+- **Most Comments**: Shows posts with highest number of comments first
+- **New Comments**: Bumps posts to the top when they receive a new reply Analogous to the sorting of traditional forums.
+- **Top Day**: Highest scoring posts during the last 24 hours
+- **Top Week**: Highest scoring posts during the last 7 days
+- **Top Month**: Highest scoring posts during the last 30 days
+- **Top Year**: Highest scoring posts during the last 12 months
+- **Top All Time**: Highest scoring posts during all time
+
+Comments are by default arranged in a hierarchy which shows at a glance who it is replying to. Top-level comments which reply directly to a post are on the very left, not indented at all. Comments that are responding to top-level comments are indented one level, and each further level of indentation means that the comment is deeper in the conversation. With this layout it is always easy to see the context for a given comment, simply scroll up to the next comment which is indented one level less.
+
+Comments can be sorted in the following ways. These all keep the indentation intact, so only replies to the same parent are shuffled around. 
+
+- **Hot** (default): Equivalent to the *Hot* sort for posts
+- **Top**: Shows comments with highest score first
+- **New**: Shows most recent comments first
+- **Old**: Shows oldest comments first
+
+Additionally there is a sort option **Chat**. This eliminates the hierarchy, and puts all comments on the top level, with newest comments shown at the top. It is useful to see new replies at any point in the conversation, but makes it difficult to see the context.
```

Comment:
@dessalines Is this explanation correct, or should something be clarified? 

I also removed the graphs and formulas, planning to move that somewhere in the developer chapter.